### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSnippets_ScreenReader: NVDA does not specify which files are New Snippets and which are Existing snippets. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52038]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2
Mac Version: HighSierra10.13.5 
Jaws Version: 18.0.5038

Repro Steps:
1. Launch VS Code while NVDA is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "User Snippets" from dropdown..

Actual:
NVDA does not specify which files are New Snippets and which are Existing snippets. 

Expected:
NVDA should announce specifically which are New snippets and which are existing .

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the proper information about User Snippets.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Problems NVDA ExistingSnippets.pptx](https://github.com/Microsoft/vscode/files/2106048/A11y.VSCode.Problems.NVDA.ExistingSnippets.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52383
